controllers/provisioners/eks/scaling/interface.go

```diff
@@ -58,6 +58,14 @@ type CreateConfigurationInput struct {
 	Volumes               []v1alpha1.NodeVolume
 	UserData              string
 	SpotPrice             string
+	LicenseSpecifications []string
+	Placement             *PlacementInput
```

Comment:
I wondering if we should just use `v1alpha1.Placement` directly instead of having another struct `PlacementInput` here?
Sane as we do for Volumes in line 58. This would also make `GetPlacementInput` redundant
WDYT?

Reply:
Sounds good, I forget where but I remember we used an internal model in one other case. The only argument I can see is that we might want to change the internal API independent of the public API.

I'll try it out with `v1alpha.PlacementSpec` and see what it looks like :)